.env.backend

```diff
@@ -0,0 +1,9 @@
+BACKEND_PORT=8080
+BACKEND_SSL=false
+
+VIDEO_FACTORY_ENDPOINT=http://192.168.101.107:8081/api/
+
+SENDGRID_API_KEY=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
```

Comment:
don't commit key like that please.

Reply:
Yes, this file was renamed and the new name was not updated into .gitignore file. 
Fix.